benchmarks/continous/db/tool/orm/src/models.rs

```diff
@@ -0,0 +1,23 @@
+use chrono::{DateTime, Utc};
+use diesel::prelude::Insertable;
+
+use crate::schema::ft_transfers;
+
+#[derive(Insertable)]
+#[diesel(table_name = ft_transfers)]
+pub struct NewFtTransfer<'a> {
+    // TODO store start and time in two separate columns
```

Comment:
If possible I'd like to first get review of the general approach in this PR and then resolve these two TODOs here in a follow-up PR.